ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -33,7 +33,7 @@ const REASON_TO_DESCRIPTION_TKEY = Object.freeze({
 
   [BlockaidReason.blurFarming]: 'blockaidDescriptionBlurFarming',
 
-  [BlockaidReason.errored]: 'blockaidDescriptionErrored', // TODO: change in i8n
+  [BlockaidReason.errored]: 'blockaidDescriptionErrored',
```

Comment:
I think we should keep above comment, the issue I think is when i18n message keys should not be used like above. That makes it hard for i18n tool to discover un-used keys etc.